src/locale/locales/fr/messages.po

```diff
@@ -76,6 +76,10 @@ msgstr "{0, plural, one {repost} other {reposts}}"
 msgid "{0, plural, one {Unlike (# like)} other {Unlike (# likes)}}"
 msgstr "{0, plural, one {Déliker (# like)} other {Déliker (# likes)}}"
 
+#: src/screens/StarterPack/StarterPackLandingScreen.tsx:218
+msgid "{0} joined this week"
+msgstr "{0} ont été inscrites cette semaine"
```

Comment:
I wasn't quite sure how to handle _écriture inclusive_ here, since it's talking about people is this OK?

Reply:
I don't think it would work in French to simply says `<number> did something` unless it's part of an enumeration of some sort (i.e. it would work only in this case: `10 personnes sont allées à la boulangerie, dont 5 qui ont acheté des baguettes, hon hon`). We have to precise what we are talking about.

So here, using either `comptes` or `personnes` would side-step the problem:

```suggestion
msgstr "{0} personnes se sont inscrites cette semaine"
```

Reply:
Makes sense to me 👌